lib/authorization.rb

```diff
@@ -21,6 +21,16 @@ def self.authorized_resources(subject_id, actions)
     matched_policies(subject_id, actions).map { _1[:tagged_id] }
   end
 
+  def self.extend_actions(actions)
```

Comment:
This is more like "expand" than "extend" right?

Reply:
Renamed